arangod/Metrics/ClusterMetricsFeature.cpp

```diff
@@ -68,73 +73,218 @@ void ClusterMetricsFeature::validateOptions(
   }
 }
 
-void ClusterMetricsFeature::start() { asyncUpdate(); }
+void ClusterMetricsFeature::start() {
+  if (wasStop()) {
+    return;
+  }
+  auto& ci = server().getFeature<ClusterFeature>().clusterInfo();
+  ci.initMetricsUpdate();
+  aql::QueryString writeText{
```

Comment:
I suggest we move initialization logic to update step, we can execute something like
```
LET docs = (FOR j IN _metrics FILTER j.version < @v AND j._key == @keyToUpdate RETURN j._key);
FOR key IN COUNT(docs) > 0 ? docs : [@keyToUpdate]
  UPSERT { _key: key }
  INSERT { _key: key, version: 0, data: [] }
  UPDATE { data: @d, version: @v } IN _metrics
```
So we don't need a dedicated initialization step.

Reply:
We need start step because of update(read)